contrib/babelfishpg_tsql/src/fts_parser.y

```diff
@@ -219,13 +248,9 @@ static char *trimInsideQuotes(char *s) {
     size_t newLength;
     bool insideQuotes;
 
-    if (s == NULL) {
-        return NULL;
-    }
-
-    length = strlen(s);
-    if (length == 0) {
-        return s; // Empty string, nothing to trim
+     // Empty string, nothing to trim
```

Comment:
it looks like trim() and trimInsideQuotes() are very similar functions, can they be combined somehow?